### PR TITLE
updated example app for automation

### PR DESCRIPTION
**Pull request by sayaansaha**
Added crash_app and load_test buttons

### CODE REVIEW

File: Leanplum-iOS-SDK-Example/Classes/ViewControllers/PushNotificationsViewController.m [hunk 1]
```diff
@@ -31,4 +31,17 @@ - (IBAction)onButtonClicked:(UIButton *)sender {
...
+    for (unsigned int i = 0; i <= 10020; i++) {
+        [Leanplum track:@"load_test"];
+        
+    }
+}
+
+- (IBAction)crash_app:(UIButton *)sender {
+    exit(0);
+}
+
```
Comment (sayaansaha) [inline Leanplum-iOS-SDK-Example/Classes/ViewControllers/PushNotificationsViewController.m:43]: These spaces are unnecessary, will remove and amend commit in the morning

File: Leanplum-iOS-SDK-Example/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -100,7 +117,7 @@
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="2fq-zs-ODj" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="2142" y="-1465"/>
+            <point key="canvasLocation" x="2141.5999999999999" y="-1465.8170914542729"/>
```
Comment (benmarten) [inline Leanplum-iOS-SDK-Example/Base.lproj/Main.storyboard:120]: this is prob auto generated, can you anyways make sure that the buttons are nicely centered/aligned :)

File: Leanplum-iOS-SDK-Example/Classes/ViewControllers/PushNotificationsViewController.m [hunk 3]
```diff
@@ -31,4 +31,17 @@ - (IBAction)onButtonClicked:(UIButton *)sender {
     [Leanplum track:@"push_notification_example"];
 }
 
+- (IBAction)load_test:(UIButton *)sender {
+    for (unsigned int i = 0; i <= 10020; i++) {
+        [Leanplum track:@"load_test"];
+
```
Comment (benmarten) [inline Leanplum-iOS-SDK-Example/Classes/ViewControllers/PushNotificationsViewController.m]: empty line

**Review by benmarten — APPROVED** [review]

File: Leanplum-iOS-SDK-Example/Base.lproj/Main.storyboard (hunk 2 above)
Comment (sayaansaha) [reply Leanplum-iOS-SDK-Example/Base.lproj/Main.storyboard:120]: It was auto generated, but I tested the app and it seemed pretty aligned.

File: Leanplum-iOS-SDK-Example/Classes/ViewControllers/PushNotificationsViewController.m (hunk 3 above)
Comment (sayaansaha) [reply Leanplum-iOS-SDK-Example/Classes/ViewControllers/PushNotificationsViewController.m]: i'll take care of the empty lines and push changes shortly

**Pull request merged**